packages/playground/windows/playground/HostingPane.xaml.cpp

```diff
@@ -467,3 +468,13 @@ void HostingPane::LoadKnownApps() {
     x_ReactAppName->SelectedIndex = 0;
   }
 }
+
+void Playground::HostingPane::OnToggleButtonClicked(
```

Comment:
Can we implement this in the debug menu?  ReactControl.cpp has a TBD for Show Inspector.

Reply:
I was looking for the debug menu it in UWP, but I couldn't find it. How do I invoke the debug menu at runtime?

Reply:
Ctrl + Shift + D

---
In reply to: [315316588](https://github.com/microsoft/react-native-windows/pull/2948#discussion_r315316588) [](ancestors = 315316588)